### ISSUE TITLE
VS Code extension error: unable to get local issuer certificate

### ISSUE THREAD

**Issue by ami-the-dev** [WordPress/playground-tools#138]
### WordPress Playground error: unable to get local issuer certificate

After installed **"WordPress Playground for VS Code**" Extension, "**Start WordPress Server**"  then getting an error or issue is "**WordPress Playground error: unable to get local issuer certificate**"  
How to fix it can anyone help me out about it.

### Other Sever can easily start without SSL as react etc
 
**Note:** Find the attachment regarding this error:
![WordPress Playground error](https://github.com/WordPress/wordpress-playground/assets/17451023/c3d9d665-2df3-42e2-afb2-2d1a6d7e8713)

**Comment by adamziel** [comment]
Thank you for reporting @HighTechGalaxy! I'm moving this issue to https://github.com/WordPress/playground-tools which is where the VS Code plugin lives.

**Comment by ami-the-dev** [comment]
@adamziel **VS Code extension error** still not fixed or no solution for this.....

**Comment by adamziel** [comment]
@HighTechGalaxy you're right. This issue is on the list, but I cannot promise any timeline for the fix. What would help, though, is if you could share which system are you on. Is it Windows? Mac? Linux?

**Comment by Eitan-brightleaf** [comment]
@adamziel I am on windows and having the same issue

**Comment by moop-moop** [comment]
We are seeing this issue in a corporate environment with an egress, decrypting firewall.
Both on Windows and MacOs, different errors, but all related to TLS, certificates, or self-signing.

**Comment by moop-moop** [comment]
I forgot to mention a corporate VPN is in the mix.
